src/globus_sdk/services/transfer/client.py

```diff
@@ -213,7 +216,9 @@ def endpoint_search(
         self,
         filter_fulltext: Optional[str] = None,
         filter_scope: Optional[str] = None,
-        **params,
+        limit: Optional[int] = None,
+        offset: Optional[int] = None,
```

Comment:
I can see why you found adding these noteworthy, but I actually think it's a good change.
It makes the SDK more verbose, but it puts more parameters directly into our documentation. I think you made the right call on how to handle this.